BLSQ/iaso
# IA-2342: Filter OrgUnits by Project

Allow to filter OrgUnitTypes by project
Allow to filter OrgUnits by project
Add "Project" select dropdown in OrgUnits' filters component

Related JIRA tickets : IA-2342

## Self proofreading checklist


## Changes

When listing the OrgUnits, there could be multiple OrgUnitTypes in the dropdown with the same name (because they belong to different projects) and we don't know which one to select. This change adds a "Project" dropdown which filters the OrgUnitTypes dropdown list but also applies to the OrgUnit list search results.

## How to test

1. Ensure you have more than one project. 
1. Ensure you have OrgUnit types linked to your projects.
1. Go to the OrgUnits list screen
1. Open the OrgUnitTypes dropdown field, see that they are all listed
1. Select a project in the Project dropdown field
1. Open the OrgUnitTypes dropdown field, see that only the OrgUnitTypes of the selected project are all listed
1. Play with the search to only display the OrgUnits you want
